front/package.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
The project already has a package.json file, so what we are going to need to do is either use this and copy over the other dependencies, or vice versa